app/src/main/kotlin/com/simplemobiletools/filemanager/pro/activities/SettingsActivity.kt

```diff
@@ -134,34 +143,39 @@ class SettingsActivity : SimpleActivity() {
     }
 
     private fun toggleShowHidden() {
-        settings_show_hidden.toggle()
-        config.showHidden = settings_show_hidden.isChecked
+        binding.settingsShowHidden.toggle()
+        config.showHidden = binding.settingsShowHidden.isChecked
     }
 
     private fun setupEnablePullToRefresh() {
-        settings_enable_pull_to_refresh.isChecked = config.enablePullToRefresh
-        settings_enable_pull_to_refresh_holder.setOnClickListener {
-            settings_enable_pull_to_refresh.toggle()
-            config.enablePullToRefresh = settings_enable_pull_to_refresh.isChecked
+        binding.apply {
+            settingsEnablePullToRefresh
```

Comment:
hm?

Reply:
didn't get the question @tibbi

Reply:
what is that line for? Seems like some error.

Reply:
ops cleanup mistake. it has no effect but yeah, i forgot to remove it.  fixed.